client/src/components/CreateSession.jsx

```diff
@@ -79,29 +88,39 @@ class Search extends React.Component {
   render () {
     return (
       <div>
-        <h2>Search</h2>
-        <form>
-          <label htmlFor="searchRestaurants">Search restaurants by name:</label>
-          <input type="text" inputMode="search" name="searchRestaurants" value={this.state.searchQuery} onChange={this.handleChange} />
-          <input type="submit" value="Search" onClick={this.getRestaurants} />
-        </form>
-
-        <div>
-          {this.state.showSuggested ? 'Suggested Restaurants Nearby:' : 'Search Results:'}
-            <ul>
-              {this.state.restaurants.map(restaurant =>
-                <li key={restaurant.restaurant_id}>
-                  <span>{restaurant.restaurant_name} - {restaurant.address.formatted}</span>
-                  <Link to='/add-guests' >
-                    <button onClick={() => this.selectRestaurant(restaurant)}>Select</button>
-                  </Link>
-                </li>
-              )}
-            </ul>
-        </div>
+        <h2>Create Session</h2>
+        <h4>Name your session and select a restaurant to get started.</h4>
+          {!this.state.sessionNameSaved
```

Comment:
just a nit, nothing to change now, but adding parentheses and indentation surrounding the ternary JSX makes it much more readable

Reply:
looks good